scripts/setup_postgres.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash
+
+# errexit. Any subsequent commands which fail will cause the
+#          shell script to exit immediately.
+set -e
+
+if [[ -z "${PROJECT_NAME}" ]]; then
+  echo "Environment variable PROJECT_NAME not found"
+  echo "For local development, are you running this script in pipenv?"
+  exit 1
+fi
+
 echo "Creating user database for $(whoami)"
 createdb $(whoami)
```

Comment:
I notice that this command returns an error if the db exists already. There's another createdb below but it's run after dropdb --if-exists so we could do the same here. But I've been thinking maybe we should assert that this script should only be run once? I don't think running this script several times is a common scenario anyways. We can clean up the dropdb lines.
Another topic is that I don't believe we use the users created in this script (I could be wrong tho). From what I understand, our DATABASE_URL uses the default user b/c username is not specified. The default user in postgres is named `whoami` so we could just stick with that user.
Thoughts on this @imruahmed ?